Sources/ArgumentParser/Parsable Properties/Flag.swift

```diff
@@ -112,6 +112,50 @@ public enum FlagExclusivity {
   case chooseLast
 }
 
+extension Flag where Value == Optional<Bool> {
+  /// Creates a Boolean property that reads its value from the presence of
+  /// one or more inverted flags.
+  ///
+  /// Use this initializer to create an optional Boolean flag with an on/off
+  /// pair. With the following declaration, for example, the user can specify
+  /// either `--use-https` or `--no-use-https` to set the `useHTTPS` flag to
+  /// `true` or `false`, respectively. If neither is specified, the resulting
+  /// flag value would be `nil`.
+  ///
+  ///     @Flag(inversion: .prefixedNo)
+  ///     var useHTTPS: Bool?
+  ///
+  /// To customize the names of the two states further, define a
```

Comment:
You can delete from here down to the parameters, since this is covered in the guide now.